SocialCareCaseViewerApi/V1/Boundary/Requests/PatchPersonRequest.cs

```diff
@@ -0,0 +1,149 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using SocialCareCaseViewerApi.V1.Domain;
+using EmailAddress = SocialCareCaseViewerApi.V1.Domain.EmailAddress;
+using KeyContact = SocialCareCaseViewerApi.V1.Domain.KeyContact;
+using GpDetails = SocialCareCaseViewerApi.V1.Domain.GpDetailsDomain;
+using PhoneNumber = SocialCareCaseViewerApi.V1.Domain.PhoneNumber;
+using LastUpdated = SocialCareCaseViewerApi.V1.Domain.LastUpdatedDomain;
+
+namespace SocialCareCaseViewerApi.V1.Boundary.Requests
```

Comment:
this would benefit from `#nullable enable` to ensure consistent and safe handling of properties

Reply:
Done